packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -421,10 +421,10 @@ export class RpcContractProvider
    */
   async transferTicket(params: TransferTicketParams) {
     if (validateAddress(params.destination) !== ValidationResult.VALID) {
-      throw new InvalidAddressError(params.destination, 'param destination');
+      throw new InvalidAddressError(params.destination, 'params destination');
```

Comment:
I notice the `errorDetail` param being passed as `params destination`

How would the error message look like when an `errorDetail` is passed?

Reply:
for now `Address "${address}" is invalid. params destination` 
I understand it's not ideal, am looking to make it `Address "${address}" is invalid. "INVALID_CHECKSUM" found in transferTicket params destination`
Will need a bigger undertaking to show why it's invalid, thinking to do that in task3

Reply:
personally I find the `params destination` bit could be a bit misleading if people don't know which part of the code throws it. I think we can omit it for now. What do you think?

Reply:
Will this help? `Address "${address}" is invalid. in transferTicket params destination`?
Was added there cuz the params can have InvalidAddressError with params source or destination
But I don't mind omit it for now to revisit them in task3

Reply:
I think displaying the PKH is clear enough to pinpoint which parameter field was invalid. 

And in cases where both source and destination are the same, the errorDetail would be a bit redundant.

Reply:
they're all removed